### PR TITLE
Ignore AI for dragging crew checks

### PR DESCRIPTION
**Pull request by PabstMirror**
#2590 

UAV / Automated Turrets have AI "crew"
The weight should be the same so we can just ignore them in the crew count check.

### CODE REVIEW

**Comment by PabstMirror** [comment]
Also, AI turrets will continue to shoot while you carry it, which is kind of awesome.

untitled

**Comment by commy2** [comment]
Wow. `disableAI` ?

**Comment by commy2** [comment]
Why does `crew` return drone logics anyway?

File: addons/dragging/functions/fnc_canDrag.sqf [hunk 1]
```diff
@@ -21,7 +21,7 @@ _target = _this select 1;
 
 if !([_unit, _target, []] call EFUNC(common,canInteractWith)) exitWith {false};
 
-// a static weapon has to be empty for dragging
-if ((typeOf _target) isKindOf "StaticWeapon" && {count crew _target > 0}) exitWith {false};
+// a static weapon has to be empty for dragging (ignore UAV AI
```
Comment (jonpas) [inline addons/dragging/functions/fnc_canDrag.sqf]: Missing `)`

**Comment by commy2** [comment]
Pulled this in https://github.com/acemod/ACE3/pull/2564

**Pull request closed without merging**